core/src/main/java/org/apache/accumulo/core/util/threads/ThreadPools.java

```diff
@@ -512,10 +516,16 @@ public Future<?> submit(Runnable task) {
    * If you need the server-side shared ScheduledThreadPoolExecutor, then use
    * ServerContext.getScheduledExecutor()
    */
+  @SuppressWarnings("deprecation")
```

Comment:
You can narrow this warnings suppression to a variable rather than the whole method by doing something like:
```java
Property prop = conf.resolve(Property.GENERAL_THREADPOOL_SIZE, Property.GENERAL_SIMPLETIMER_THREADPOOL_SIZE);
```
and then put the warnings suppression only on that variable, then use that variable in the next call to createExecutorService.

Reply:
I will do this when Exception Handling is no longer required. 
Otherwise, it only serves to clutter the code.

Reply:
I did it in 58e8ec5b960dd2f6a4c20b4b0c2212ee67b0e5e6